spec/features/schemes_spec.rb

```diff
@@ -149,20 +149,45 @@
         context "when I click to see individual scheme" do
           let(:scheme) { schemes.first }
 
-          before do
-            click_link(scheme.service_name)
+          context "when there is a secondary client group" do
+            before do
+              click_link(scheme.service_name)
+            end
+
+            it "shows me details about the selected scheme" do
+              expect(page).to have_content(schemes.first.id_to_display)
+              expect(page).to have_content(schemes.first.service_name)
+              expect(page).to have_content(schemes.first.sensitive)
+              expect(page).to have_content(schemes.first.scheme_type)
+              expect(page).to have_content(schemes.first.registered_under_care_act)
+              expect(page).to have_content(schemes.first.primary_client_group)
+              expect(page).to have_content(schemes.first.has_other_client_group)
+              expect(page).to have_content(schemes.first.secondary_client_group)
+              expect(page).to have_content(schemes.first.support_type)
+              expect(page).to have_content(schemes.first.intended_stay)
+            end
           end
 
-          it "shows me details about the selected scheme" do
-            expect(page).to have_content(schemes.first.id_to_display)
-            expect(page).to have_content(schemes.first.service_name)
-            expect(page).to have_content(schemes.first.sensitive)
-            expect(page).to have_content(schemes.first.scheme_type)
-            expect(page).to have_content(schemes.first.registered_under_care_act)
-            expect(page).to have_content(schemes.first.primary_client_group)
-            expect(page).to have_content(schemes.first.secondary_client_group)
-            expect(page).to have_content(schemes.first.support_type)
-            expect(page).to have_content(schemes.first.intended_stay)
+          context "when there is no secondary client group" do
+            before do
+              scheme.has_other_client_group = "No"
+              scheme.secondary_client_group = nil
+              scheme.save!
```

Comment:
Just a bit of code golf, but would this work? 
```suggestion
              scheme.update!(has_other_client_group: "No", secondary_client_group: nil)
```